pages/api/collection/index.ts

```diff
@@ -30,7 +30,7 @@ export default withAuth({
     logger.error(err);
 
     return res.status(500).json({
-      error: 'Error creating collection',
+      error: 'Error: Name is longer than the maximum allowed length (50 characters)',
```

Comment:
So this error won’t always be because of a too long name. Can we change the check to be front end not back end?

Reply:
Good point, in the onSubmit function we can add some validation checks, then show a toast error and exit early if we find any issues